sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java

```diff
@@ -159,23 +167,64 @@ public static void attestAsync1() {
         // END: com.azure.security.attestation.AttestationAsyncClient.getAttestationSignersWithResponse
     }
 
-    public static void setPolicyCheckHash() {
-        AttestationAdministrationAsyncClient client = null;
+    public static void setPolicyCheckHashAsync() {
+        String endpoint = System.getenv("ATTESTATION_AAD_URL");
```

Comment:
We seem to be using endpoint ""[https://attestcus.cus.attest.azure.net](https://attestcus.cus.attest.azure.net)" and System.getenv("ATTESTATION_AAD_URL"); interchangeably. Should we use a consistent one?

Reply:
It's actually "https://sharedcus.cus.attest.azure.net" and that's an example of a shared attestation instance. ATTESTATION_AAD_URL is a customer provided attestation url. Both are valid and different uses and I want to make sure I have snippets for both.

And the reason I'm using Central US is that there are some bug fixes being deployed to the service instances, and the instances in West US haven't been updated yet (but Central US has the bug fixes).

Reply:
Sounds good.